data/onvif_users.htdigest

```diff
@@ -0,0 +1 @@
+admin:Onvif_service:bfa1427cd1a06d1120c1e85900c9e434
```

Comment:
@AkshayPS12 is there a test which shows the password stored here ?

Reply:
no just HT digest parsing , password is showed no where..